[2022-08-19 13:05:04,262] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-19T13:00:00+00:00 [queued]>
[2022-08-19 13:05:04,307] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-19T13:00:00+00:00 [queued]>
[2022-08-19 13:05:04,307] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 13:05:04,307] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 13:05:04,307] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 13:05:04,377] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2022-08-19 13:00:00+00:00
[2022-08-19 13:05:04,386] {standard_task_runner.py:52} INFO - Started process 68324 to run task
[2022-08-19 13:05:04,393] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2022-08-19T13:00:00+00:00', '--job-id', '522', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpg7q7p1kq', '--error-file', '/tmp/tmp24ld29yk']
[2022-08-19 13:05:04,395] {standard_task_runner.py:80} INFO - Job 522: Subtask popula_elastic_finalizados
[2022-08-19 13:05:04,625] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-19T13:00:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-19 13:05:05,096] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2022-08-19T13:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-19T13:00:00+00:00
[2022-08-19 13:05:05,117] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.018s]
[2022-08-19 13:05:09,008] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_finalizados [status:200 request:0.001s]
[2022-08-19 13:05:09,457] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-19 13:05:09,664] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.071s]
[2022-08-19 13:05:09,859] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.138s]
[2022-08-19 13:05:10,633] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.538s]
[2022-08-19 13:05:11,567] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.852s]
[2022-08-19 13:05:11,816] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.670s]
[2022-08-19 13:05:12,448] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.572s]
[2022-08-19 13:05:13,233] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.719s]
[2022-08-19 13:05:13,483] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.573s]
[2022-08-19 13:05:13,643] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.254s]
[2022-08-19 13:05:13,685] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.150s]
[2022-08-19 13:05:13,783] {logging_mixin.py:115} INFO - 0       19-08-2022
1       19-08-2022
2       19-08-2022
3       19-08-2022
4       19-08-2022
           ...    
4653    20-07-2022
4654    20-07-2022
4655    20-07-2022
4656    20-07-2022
4657    20-07-2022
Name: DataFinalizacao, Length: 4658, dtype: object
[2022-08-19 13:05:13,785] {python.py:173} INFO - Done. Returned value was: None
[2022-08-19 13:05:13,813] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20220819T130000, start_date=20220819T130504, end_date=20220819T130513
[2022-08-19 13:05:13,921] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 13:05:14,011] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
